JobConfig/digitize/DTSfilter.fcl

```diff
@@ -0,0 +1,38 @@
+#
+#  to filter DTS
+#
+#include "Production/JobConfig/common/prolog.fcl"
+#include "Production/JobConfig/digitize/prolog.fcl"
+# name all processes the same, to mask any provenance in ensemble creation
+process_name: Digitize
```

Comment:
this process name should be DTSfilter.  If we rename this script, the process_name should be updated too